terraform-modules/aws/cloudposse/aws-cloudtrail-cloudwatch-alarms/main.tf

```diff
@@ -21,7 +21,7 @@ module "kms_cloudtrail" {
 }
 
 module "cloudtrail_s3_bucket" {
-  source  = "github.com/ManagedKube/terraform-aws-cloudtrail-s3-bucket.git//?ref=0.24.0"
+  source  = "github.com/ManagedKube/terraform-aws-cloudtrail-s3-bucket.git//?ref=0.25.0"
```

Comment:
This was generated by you @sekka1 in the following release: https://github.com/ManagedKube/terraform-aws-cloudtrail-s3-bucket/releases/tag/0.25.0